packages/ui-stencil/src/components/internal/orama-chat-suggestions/orama-chat-suggestions.tsx

```diff
@@ -31,6 +31,7 @@ export class OramaChatSuggestions {
           return (
             <li key={suggestion} class={`suggestion-item-${classSuffix}`}>
               <button
+                focus-on-arrow-nav
```

Comment:
is this attr intended?

Reply:
yes, it's used to specify which elements you can navigate (focus) using the arrow up/down keys. 
users can select a suggestion by using keyboard navigation, it affects only "search" context mode

Reply:
got it, so you're basically replacing
`'a[href], button, textarea, input, select, [tabindex]:not([tabindex="-1"])',`
with 
`focus-on-arrow-nav`
as a selector, to explicitly select elements you can navigate via keyboard.

The first selector ensured that all native interactive elements (present and future) could be selected via kyb nav (that is the expected), did you encounter any issues with the previous selector? You were not able to select the suggestion before?

Reply:
Thanks for clarifying, got the point! Agree with the more controlled approach over arrow up/down.